Assets/animController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class animController : MonoBehaviour
+{

```

Comment:
I do a short bracket style, not super common in C# but I prefer it so move these down e.g.:
```C#
public class AnimController : Monobehviour {
...
}
```